cluster/selfplay/cc-Dockerfile

```diff
@@ -46,12 +52,12 @@ RUN mkdir /bazel && \
 
 ENV CI_BUILD_PYTHON python3
 
+run mv /usr/bin/python /usr/bin/python.bak
```

Comment:
this is weird

Reply:
this is related to the bit you did with the different pythons in your minigui dockerfile.  solved the same problem just a different way.